src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -943,6 +944,29 @@ registerThemingParticipant((theme: ITheme, collector: ICssStyleCollector) => {
 		`);
 	}
 
+	const editorGroupHeaderTabsBackground = theme.getColor(EDITOR_GROUP_HEADER_TABS_BACKGROUND);
+	const editorGroupBackground = theme.getColor(EDITOR_GROUP_BACKGROUND);
+	const editorBackgroundColor = theme.getColor(editorBackground);
+	const editorDragAndDropBackground = theme.getColor(EDITOR_DRAG_AND_DROP_BACKGROUND);
+	const themeNotHC = theme.type !== 'hc';
+	let workbenchBackground: Color;
+	let adjustedTabBackground: Color;
+	let adjustedTabDragBackground: Color;
+
+	switch (theme.type) {
```

Comment:
@Dari-K this seems at the wrong place, can we move it into `theme.ts` e.g. where the knowledge belongs to?